extra/intl-extra/Tests/Fixtures/format_date.test

```diff
@@ -19,6 +21,8 @@ Aug 7, 2019, 11:39:12 PM
 23:39
 07/08/2019
 mercredi 7 août 2019 à 23:39:12 temps universel coordonné
+07/08/2019
+mercredi 7 août 2019 à 23:39:12 temps universel coordonné
```

Comment:
This looks suspicious to me as I don't see anything "relative" here. Does it mean that it's not implemented for French? We should find an example that returns something different.

Reply:
It's limited to `yesterday`, `today`, and `tomorrow`. We need a close date to see a difference.